Redesigned the model component.

goodguysarvagya opened the pull request:
This is my first PR, hope i don't break anything.

goodguysarvagya reviewed (commented):
I don't know what is this probably prettier kicked in and did some fixing.

goodguysarvagya commented:
@AndriyMulyar I have also redesigned some other components as result of those the lighthouse score went up by a lot feedback is appreciated. I will create further PRs with these fixes as per your guidance.

![lighthouse_score_difference](https://github.com/nomic-ai/gpt4all.io/assets/87168264/ab0c8071-ec57-4b79-a550-a5a126946444)

AndriyMulyar commented:
Really nice! Will review soon.

AndriyMulyar commented on frontend/gpt4all.io/src/components/models.jsx:
```diff
@@ -1,88 +1,72 @@
-}
+  //const response = await fetch("https://gpt4all.io/models/models.json");
+  const response = await fetch(
+    "https://raw.githubusercontent.com/nomic-ai/gpt4all/main/gpt4all-chat/metadata/models.json"
```
I want to retain this url: https://gpt4all.io/models/models.json

It redirects to the one you posted.

AndriyMulyar commented:
Can you link a video of the new UI

goodguysarvagya commented:
@AndriyMulyar I will upload the video asap. Thanks for the Quick PR Review. Much appreciated.

goodguysarvagya commented:
> Can you link a video of the new UI

-  Here is a demo clip i changed the gifs to mp4 videos (smaller size) and rewrote the capability card component for that and performance table to have sorting although now i see i should place an indicator to show that the table is sortable.

> https://drive.google.com/file/d/18W1GDXcI4C4J9Lv7JJ4_6gu4Ux9PDXxY/view?usp=sharing
